playground/backend/containers/go/Dockerfile

```diff
@@ -63,10 +63,27 @@ RUN mkdir /opt/mitmproxy &&\
     wget https://snapshots.mitmproxy.org/7.0.4/mitmproxy-7.0.4-linux.tar.gz &&\
     tar -zxvf mitmproxy-7.0.4-linux.tar.gz &&\
     mkdir /usr/local/share/ca-certificates/extra
-COPY entrypoint.sh /
 COPY allow_list_proxy.py /opt/mitmproxy/
 COPY allow_list.py /opt/mitmproxy/
 ENV HTTP_PROXY="http://127.0.0.1:8081"
 ENV HTTPS_PROXY="http://127.0.0.1:8081"
 
-ENTRYPOINT ["/entrypoint.sh"]
+COPY entrypoint.sh /
+COPY proxy.sh /
+# Run proxy
+RUN chmod +x /proxy.sh
+RUN /proxy.sh
+
+# Create a user group userGroup
+RUN groupadd --gid 20000 appuser \
+  && useradd --uid 20000 --gid appuser --shell /bin/bash --create-home appuser
+# Chown all the files to the app user.
+RUN chown -R appuser:appuser /opt/playground/
+RUN chown -R appuser:appuser /opt/playground/backend/
+RUN chown -R appuser:appuser /opt/playground/prepared_folder/
+RUN chmod +x /entrypoint.sh
+
+# Switch to appuser
+USER appuser
+
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
Could you please add an empty line at the end of the file?

Reply:
Done.